### DISCUSSION TITLE
Working EFIS/DSP page

### DISCUSSION THREAD

**Discussion by OldGroof** [magknight/787aviatorsTracker#526] [Ideas]
**Is your feature request related to a problem? Please describe.**
At the moment it's been disabled but it'd be nice to have it partially working with the things that aren't possible still disabled

**Describe the solution you'd like**
Have the MFD part working where you can select the MFD's and what to display, the MINS and BAROSET areas working and maybe the map options as well.

**Comment by aeroniemi** [comment]
page activation is intended to be added once full mfd mode status is available

**Comment by aeroniemi** [comment]
could probably make this into a thing with ND positions fixed - needs updating to current ruleset

**Comment by aeroniemi** [comment]
easy to make lower segment work, much more difficult to make the top side work as we don't have a mfd display scratchpad system (it's not used by any other displays except this and maintinence/CB, which won't be added). Tempted just to redesign the top part to have command-operated buttons rather than keyboard entry

**Reply by aeroniemi** [reply]
it could be done now, but not sure it's worth actually doing
